src/pages/MetricDetail.svelte

```diff
@@ -3,9 +3,54 @@
 
   export let params;
   const metricDataPromise = getMetricData(params.app, params.metric);
+
+  function getMetricDocumentationURI(type) {
+    const sourceDocs = "https://mozilla.github.io/glean/book/user/metrics/";
+    const links = {
+      memory_distribution: `${sourceDocs}memory_distribution.html`,
+      quantity: `${sourceDocs}quantity.html`,
+      custom_distribution: `${sourceDocs}custom_distribution.html`,
+      string_list: `${sourceDocs}string_list.html`,
+      labeled_string: `${sourceDocs}labeled_strings.html`,
+      timespan: `${sourceDocs}timespan.html`,
+      datetime: `${sourceDocs}datetime.html`,
+      string: `${sourceDocs}string.html`,
+      timing_distribution: `${sourceDocs}timing_distribution.html`,
+      boolean: `${sourceDocs}boolean.html`,
+      labeled_counter: `${sourceDocs}labeled_counters.html`,
+      uuid: `${sourceDocs}uuid.html`,
+      counter: `${sourceDocs}counter.html`,
+      event: `${sourceDocs}event.html`,
+      jwe: `${sourceDocs}jwe.html`,
+    };
+
+    return links[type] || sourceDocs;
```

Comment:
This seems like a good approach to me, but there's no need to define the sourceDocs variable each time-- instead, try something like this:

```suggestion
    return `${sourceDocs}${links[type]}` || sourceDocs;
```